### ISSUE TITLE
Upgrade win32 dependency to latest

### ISSUE THREAD

**Issue by BirjuVachhani** [BringingFire/rich_clipboard#17]
I am using [file_picker](https://pub.dev/packages/file_picker) package along with rich_clipboard and I am not able to build it with because of this error.

```
Because no versions of rich_clipboard match >1.0.1 <2.0.0 and rich_clipboard 1.0.1 depends on rich_clipboard_windows ^1.0.0, rich_clipboard ^1.0.1 requires rich_clipboard_windows ^1.0.0.
And because every version of rich_clipboard_windows depends on win32 ^2.5.1 and file_picker >=5.1.0 depends on win32 ^3.0.0, rich_clipboard ^1.0.1 is incompatible with file_picker >=5.1.0.
```

**Comment by rodrigo-badaro** [comment]
I'm with the same problem, you can try use file_picker in version 5.0.0 for this moment

**Comment by jmatth** [comment]
Thanks for the report. I just published version 1.0.2 of `rich_clipboard_windows` to pub.dev, which should fix the issue.
